packages/strapi-design-system/src/Alert/__tests__/Alert.spec.js

```diff
@@ -5,12 +5,29 @@ import { Alert } from '../Alert';
 import { ThemeProvider } from '../../ThemeProvider';
 import { lightTheme } from '../../themes';
 
-jest.mock('@strapi/icons', () => ({
-  AlertInfoIcon: () => <span>AlertInfoIcon</span>,
-  AlertSucessIcon: () => <span>AlertSucessIcon</span>,
-  AlertWarningIcon: () => <span>AlertWarningIcon</span>,
-  CloseAlertIcon: () => <span>CloseAlertIcon</span>,
-  ExternalLink: () => <span>ExternalLink</span>,
+jest.mock('@strapi/icons/AlertInfoIcon', () => ({
```

Comment:
Why do we need to mock it?

Reply:
Because of peer dependencies. react is not defined in the strapi icons since the host application should resolve react by itself. Right now, I don't know why it's a problem for jest to resolve react as a peer dependency, but that's a problem I'll check later since we used to mock icons anyway before this PR